storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -9,6 +9,42 @@
 This page documents all of the command-line options for the
 :program:`xtrabackup` binary.
 
+You invoke |program| with either the :option:`--backup` or :option:`--prepare`
+parameter.
```

Comment:
there are 4modes: `--backup`, `--prepare`, `--copy-back/--move-back` and `--stats`